app/data/1_0001.json

```diff
@@ -0,0 +1,5127 @@
+                                    ],
+                                    "description": "",
+                                    "id": "general-business-information-question-4",
+                                    "number": "1.2",
+                                    "title": "Did any of the following significant events or changes occur to {{respondent.ru_name}}?",
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "General Business Information"
+                        }
+                    ],
+                    "type": "questionnaire"
+                },
+                {
+                    "id": "general-business-information-block-5",
+                    "sections": [
+                        {
+                            "description": "<p>You have successfully completed this section</p><p>The next section covers changes in <em>business strategy and practices</em>, for example, implementing changes to marketing concepts or strategies.</p>",
+                            "id": "general-business-information-section-5",
+                            "number": "1",
+                            "questions": [
+                                {
+                                    "answers": [],
+                                    "description": "",
+                                    "id": "general-business-information-question-5",
+                                    "title": "",
+                                    "type": "General"
+                                }
+                            ],
+                            "title": "General Business Information"
+                        }
+                    ],
+                    "type": "interstitial"
+                }
+            ],
+            "id": "general-business-information-section-5-group-5",
+            "title": "General Business Information"
+        },
+        {
+            "blocks": [
+                {
+                    "id": "business-strategy-and-practices-block-6",
+                    "sections": [
+                        {
+                            "description": "<p>3 year period 1 January 2014 to 31 December 2016.</p>",
+                            "id": "business-strategy-and-practices-section-6",
+                            "number": "2",
+                            "questions": [
+                                {
+                                    "answers": [
+                                        {
+                                            "description": "For example first use of supply chain management, business re-engineering, knowledge management, lean production or quality management",
+                                            "id": "business-strategy-and-practices-answer-6-0",
+                                            "label": "New <em>business practices</em> for organising procedures",
```

Comment:
The `<em>` tags get rendered as text:
![image](https://cloud.githubusercontent.com/assets/1754822/21853651/dfcc103c-d80f-11e6-8c18-3d19718f8c80.png)